models/edr/alerts/failed_tests.sql

```diff
@@ -0,0 +1,28 @@
+{{
```

Comment:
Why do we need this view here? this can be in the CLI in the alerts model no? I think this also means new CLI version won't work with old package version

Reply:
Basically the same as [this](https://github.com/elementary-data/elementary/pull/137/files#r926585866).